src/pages/Payment/component/ShippingAddress.scss

```
@@ -16,9 +16,14 @@
     padding-bottom: 24px;
     @include head-3;
   }
-  .same-as-orderer:hover {
-    opacity: 0.5;
-    cursor: pointer;
+  .button-box {
+    display: flex;
+    .button-user-same {
+      .same-as-orderer:hover {
```

```suggestion
      .same-as-orderer
      &:hover {
```

sass에서 가상클래스는 &(parent selector)를 사용합니다
자세한내용은 [공식문서](https://sass-lang.com/documentation/style-rules/parent-selector)를 참고 해 보세요